cli/config/configuration-clh.toml.in

```diff
@@ -68,6 +68,11 @@ virtio_fs_cache_size = @DEFVIRTIOFSCACHESIZE@
 # cloud-hypervisor prefers virtiofs caching (dax) for performance reasons
 virtio_fs_cache = "always"
 
+# Block storage driver to be used for the hypervisor in case the container
+# rootfs is backed by a block device. This is virtio-scsi, virtio-blk
+# or nvdimm.
+block_device_driver = "virtio-blk"
```

Comment:
@likebreath Is the plan to just support blk and skip scsi going forward with clh?

Reply:
@amshinde I think that's right. AFAIK, there is no plan to add support for virtio-scsi in CLH in the near future. Will this feature be very useful to Kata?

Reply:
@likebreath Thats fine, I am fine with just having virtio-blk with clh.